python_modules/libraries/dagster-dask/tox.ini

```diff
@@ -19,6 +19,8 @@ commands =
   !windows: /bin/bash -c '! pip list --exclude-editable | grep -e dagster -e dagit'
   coverage erase
   echo -e "--- \033[0;32m:pytest: Running tox tests\033[0m"
+  python -m pip install "dask[dataframe]" --upgrade
```

Comment:
hey @DavidKatz-il I think these actually should move to `setup.py`, since you you import `dask.dataframe` in the package code here [1]. so I think `setup.py#33` should read:

```
'dask[dataframe]>=1.2.2',
```

Reply:
can move pyarrrow there as well